docs/release-notes/2.40.0.md

```diff
@@ -0,0 +1,111 @@
+
+# Percona Monitoring and Management 2.40.0
+
+
+| **Release date:** | [Oct 02, 2023]                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+<!---
+
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
+--->
+
+## Release Highlights
+
+### Inventory improvements
+
+#### Ability to update labels for existing services to PMM  
+
+Starting with PMM 2.40.0, editing labels for existing services is easier and much more efficient. You can edit a service label directly from the PMM UI without removing and re-adding it. For more information on editing labels, see [documentation](https://docs.percona.com/percona-monitoring-and-management/details/dashboards/dashboard-inventory.html#editing-labels-for-a-service).
+
+  ![!](../../_images/PMM_access_edit_labels.png)
```

Comment:
I think I'm getting 40 for this image

Reply:
This image is used in another PR. so when all the PR's are merged the image will be visible.